hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/NNThroughputBenchmark.java

```diff
@@ -1348,7 +1363,7 @@ class ReplicationStats extends OperationStatsBase {
     static final String OP_REPLICATION_USAGE = 
         "-op replication [-datanodes T] [-nodesToDecommission D] " +
         "[-nodeReplicationLimit C] [-totalBlocks B] [-blockSize S] "
-        + "[-replication R]";
+        + "[-replication R] [-baseDirName N]";
```

Comment:
somewhere us baseDirName D and here it is baseDirName N
Shouldn't be inconsistent I believe